src/docs/arch-hs.dox

```diff
@@ -252,12 +253,12 @@ visibility based on transaction and timestamp based visibility rules. In case th
 visible in the history store table, \c WT_NOTFOUND error is returned to the reader.
 
 @section arch_hs_rts History store and rollback-to-stable
-Rollback-to-stable searches the history store table for valid updates that are stable according to
```

Comment:
It is simply "rollback to stable" (no hyphens) on that page and elsewhere.  Could you fix it in this page please?

Reply:
Done.  Fixed all of the "rollback-to-stable" appearances on this page and grep'ed the rest of the architecture guide to identify others.